juju/juju
# feat(machine): add machine cloud instance watcher

This patch adds a simple watcher for the machine cloud instance table
that will be recruited in the uniter worker.


## Checklist




## QA steps

This watcher is not wired-up yet, so unit tests should pass:

```
TEST_PACKAGES="./domain/machine/... -count=1 -race -check.vv" make run-go-tests
```


## Links

**Jira card:** JUJU-6284
